src/main/scala/mesosphere/marathon/MarathonSchedulerService.scala

```diff
@@ -421,4 +417,18 @@ class MarathonSchedulerService @Inject() (
       reconciliationInitialDelay.toMillis + reconciliationInterval.toMillis
     )
   }
+
+  private def startAbdication(abdicationCommand: () => Unit, driverHandlesAbdication: Boolean): Unit = {
```

Comment:
I agree, ```abdicateAfterFailure``` definitely sounds better. 

Actually, this flag initially set to false, and this allows us to handle migration failure, driver creation failure and failure in prepareStart method. Maybe this flag deserves to stay here?